<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MnemonicDialog</class>
 <widget class="QDialog" name="MnemonicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Mnemonic Seed</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="btnImport">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mnemonicLabel">
     <property name="text">
      <string>Mnemonic</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTextEdit" name="tbxMnemonic">
     <property name="lineWrapMode">
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>124</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>124</height>
      </size>
     </property>
     <property name="title">
      <string>New Mnemonic</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QSpinBox" name="spinEntropy">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>40</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>16</number>
      </property>
      <property name="maximum">
       <number>64</number>
      </property>
      <property name="value">
       <number>32</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGenerate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bytes of Entropy</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbxLanguage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>French</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Japanese</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spanish</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chinese_s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chinese_t</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="11" column="0" rowspan="2" colspan="3">
    <layout class="QFormLayout" name="mnemonicLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtPassword"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="importChainLabel">
       <property name="text">
        <string>Import Chain</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chkImportChain">
       <property name="toolTip">
        <string>Only needed if you're importing </string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ensure you backup your mnemonic and password - they are not recoverable.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Warning</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
